wandb/sdk/wandb_run.py

```diff
@@ -681,6 +682,15 @@ def _init(
                 launch_config, user="launch", _allow_val_change=True
             )
 
+        # if run is from a launch queue, add queue id to _wandb config
+        launch_queue_uri = wandb.env.get_launch_queue_uri()
+        if launch_queue_uri:
+            config[wandb_key]["launch_queue_uri"] = launch_queue_uri
+
+        launch_run_queue_item_id = wandb.env.get_launch_run_queue_item_id()
+        if launch_run_queue_item_id:
+            config[wandb_key]["launch_run_queue_item_id"] = launch_run_queue_item_id
```

Comment:
might be better to tie these together somehow? We **must** have both in order for the streamtable transactions to work properly, if either is not present we don't log transactions....